server/cmwell-cons/app/cmw.env

```diff
@@ -2,6 +2,12 @@ export PATH=`pwd`/components-extras/scala/bin:$PATH
 export PATH=`pwd`/components-extras/java/bin:$PATH
```

Comment:
Add also this line inside the condition below (for completeness - not mandatory).